packages/shared/src/components/post/AuthorOnboarding.tsx

```diff
@@ -27,7 +27,7 @@ export function AuthorOnboarding({
         )}
         style={{ gridTemplateColumns: 'repeat(2, max-content)' }}
       >
-        <FeatherIcon />
+        <FeatherIcon size="xlarge" className="text-theme-status-help icon" />
```

Comment:
I think this one needs to be filled right?

Reply:
This PR is still in draft, but I will reach to Tsahi about guideline for filled icons and double check all of them